### ISSUE TITLE
Why the CrashProbe.com overview shows there are 44 cases, but there are only 22 case in the repo?

### ISSUE THREAD

**Issue by neonio** [bitstadium/CrashProbe#8]
I notice that CrashProbe.com overview section shows there are 44 cases . But when I check the git repo, there are only 22 case in the project. 

Is it because you regard different architecture (ARMv7 and ARM64) as two different condition so double the cases?

**Comment by ghost** [comment]
That is correct. The results on 32Bit and 64Bit can be very different because of the CPU architecture and compiler differences.
